lib/generators/boring/vcr/install/install_generator.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Boring
+  module Vcr
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds VCR to the application"
+      source_root File.expand_path('templates', __dir__)
+
+      class_option :test_framework,
+                   type: :string,
+                   alias: "-f",
+                   default: "rspec",
+                   enum: %w[rspec minitest],
+                   desc: "Tell us which test framework you are using. Default to rspec"
+
+      class_option :stubbing_libraries,
+                   type: :array,
+                   alias: "-sl",
+                   default: ['webmock'],
+                   enum: %w[webmock typhoeus faraday excon],
+                   desc: "Tell us stubbing library you want to use seprated by space. Default to webmock"
+
+      def add_vcr_gem
+        say "Adding VCR gems to Gemfile", :green
+        # TODO: Use check_and_install_gem method when it is available
+        gem 'vcr', group: :test
+      end
+
+      def add_stubbing_library_gems
+        say "Adding stubbing library gems to Gemfile", :green
+
+        options[:stubbing_libraries].uniq.each do |stubbing_library|
+          # TODO: Use check_and_install_gem method when it is available
+          gem stubbing_library, group: :test
+        end
+      end
+
+      def setup_vcr_for_rspec
+        test_framework = options[:test_framework]&.to_s || 'rspec'
+        return unless test_framework == 'rspec'
+
+        say "Setting up VCR for RSpec", :green
+
+        @stubbing_libraries = format_stubbing_libraries
+
+        template("vcr.rb", "spec/support/vcr.rb")
+
+        inject_into_file "spec/rails_helper.rb",
+                         "require 'support/vcr'\n\n",
+                         before: /\A/
+      end
+
+      def setup_vcr_for_minitest
+        return unless options[:test_framework].to_s == 'minitest'
+
+        say "Setting up VCR for Minitest", :green
+
+        vcr_config = <<~RUBY
+          require "vcr"
+
+          VCR.configure do |c|
+            c.cassette_library_dir = "test/vcr"
+            c.hook_into #{format_stubbing_libraries}
+            c.ignore_localhost = true
+          end
+        RUBY
+
+
+        inject_into_file "test/test_helper.rb",
+                         vcr_config, 
+                         end: /^end\s*\Z/m
+      end
+
+      private
+
+      def format_stubbing_libraries
+        options[:stubbing_libraries]
```

Comment:
Can be updated to:

`options[:stubbing_libraries].map(&:to_sym).join(", ")`

Reply:
it will not work, as we are using it on template even `:name` will be converted to string "name" but we need ":name"

Reply:
That's exactly what map(&:to_sym) is for though, did you try it?

Reply:
yes dai, I have tried it. as when we do  to_sym it will convert "string" to :string but when it is rendered on template i guess it convert it back to string. as we know :string.to_s will be "string" not ":string"

Reply:
That makes sense, then what you have done is correct.